symfony/src/Controller/DeceasedReportsController.php

```diff
@@ -109,6 +111,30 @@ public function deceasedReportNew(Request $request, ParticipantSummaryService $p
         ]);
     }
 
+    /**
+     * @Route("/stats", name="deceased_report_stats")
+     */
+    public function getStats(SessionInterface $session, DeceasedReportsService $deceasedReportsService)
+    {
+        $organizationId = $session->get('siteOrganizationId');
+        $cache = new DatastoreAdapter(100);
+        $pendingReportCountCache = $cache->getItem('deceased_reports.pending.count');
+        if ($cacheHit = $pendingReportCountCache->isHit()) {
+            $pendingReportCount = (int) $pendingReportCountCache->get();
+        } else {
+            $pendingReportCountCache->expiresAfter(60);
+            $reports = $deceasedReportsService->getDeceasedReports($organizationId, 'preliminary');
+            $pendingReportCount = count($reports);
+            $pendingReportCountCache->set($pendingReportCount);
+            $cache->save($pendingReportCountCache);
+        }
+
+        return $this->json([
+            'pending' => $pendingReportCount,
+            'cacheHit' => $cacheHit
```

Comment:
Where are we using this?

Reply:
This was mostly for debugging purposes. I typically pass along something like this when working with a cached endpoint if I need to demonstrate that a cache "miss" is happening.